cmd/svcat/command/base.go

```diff
@@ -0,0 +1,16 @@
+package command
+
+// Base of all svcat commands.
+type Base struct {
```

Comment:
I don't understand why we need this type - couldn't we just use the Context type?